src/safir/slack/sentry.py

```diff
@@ -0,0 +1,20 @@
+"""Helpers for Sentry support in SlackExceptions."""
+
+from dataclasses import dataclass
+from typing import Any
+
+__all__ = ["SentryEventInfo"]
+
+
+@dataclass
+class SentryEventInfo:
+    """A collection of metadata to add to Sentry events.
+
+    The overrideable `~SlackException.to_sentry` method returns an object of
+    this type.
+    """
+
+    tags: dict[str, str]
+    contexts: dict[str, dict[str, Any]]
+    attachments: dict[str, Any]
+    username: str | None
```

Comment:
Needs docstrings for the members (great spot to put a short version of the hints from the documentation for when to use a tag vs. a context vs. an attachment). Also, per above, default values for tags, contexts, and attachments, and maybe of `None` for username as well, seem appropriate.